packages/styles/src/tokens/_external.scss

```diff
@@ -0,0 +1,8 @@
+@use '@swisspost/design-system-tokens/channel' as channel;
+
+@use './core';
+@use './../mixins/tokens';
+
+:root {
```

Comment:
To separate these semantic tokens from the rest, we could put them in a `:scope` selector instead of `:root`.
> When used at the root level of a stylesheet, :scope is equivalent to :root, which in a regular HTML document matches the <html> element.

Source: [mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/:scope)

Reply:
While I don't see an immediate benefit, this could be nice in combination with `@scope` in the future. Not sure though if there is an actual use case. As long as we don't need to use this, I would avoid implementing it as not to create unexpected and hard to fix issues when suddenly someone uses `@scope` without thinking about the tokens.